### DISCUSSION TITLE
Is there a way to prevent any images, text/font from being brightened/darkened on Chrome?

### DISCUSSION THREAD

**Discussion by SunnyBroski** [darkreader/darkreader#8155] [Q&A] [answered]
Just recently my dark reader decided to brigthen/darken images and fonts in tandem with the contrast and brightness scroll bars. 
Just over a week ago I had ocular surgery and this stuff is really starting to bug me. Is there anyway to solve this or perhaps a way to revert it? I have no experience with code of dev tool managment so I hope that there's an easy fix to this as I am unable to use majority of my day-to-day websites. 
![image](https://user-images.githubusercontent.com/99124303/152680157-c27a3f27-1119-44f7-969d-fc62a83d380c.png)
![image](https://user-images.githubusercontent.com/99124303/152680301-9fb1648a-c1f0-41fe-87f3-6c0eaf805633.png)

**Comment by Gusted** [comment, marked as answer]
See https://github.com/darkreader/darkreader/issues/8145

**Reply by SunnyBroski** [reply]
So how do I revert those new elements? Having trouble working through all the threads

**Reply by SunnyBroski** [reply]
I'm really all new to this sorta stuff so I'm not sure how to work it all

**Reply by Gusted** [reply]
It will be fixed in the next version, which would be in the store shortly.

**Reply by SunnyBroski** [reply]
Thanks, really appreciate your help.

**Comment by siubb** [comment]
If anyone need the fix urgently, they could try to load the ver v4.9.45 as unpacked extension.
It works for me.
Remember to export / backup the settings from the stored-installed dark reader first.
